src/main/java/com/epam/aidial/core/upstream/UpstreamRoute.java

```diff
@@ -6,25 +6,26 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.Objects;
-import java.util.concurrent.atomic.AtomicLong;
 
 @RequiredArgsConstructor
 public class UpstreamRoute implements Iterator<Upstream> {
```

Comment:
Please add javadoc for the fields. They require some explanation.

Reply:
Added.